jax/_src/random.py

```diff
@@ -2688,46 +2688,55 @@ def binomial(
     n: RealArray,
     p: RealArray,
     shape: Shape | None = None,
-    dtype: DTypeLikeFloat = float,
+    dtype: DTypeLikeFloat | DTypeLikeInt = float,
 ) -> Array:
-  r"""Sample Binomial random values with given shape and float dtype.
```

Comment:
Please preserve the two-space indentation. It conforms to the style of the rest of the package, and makes diffs easier to see.